frontend/views/components/LinkToCopy.vue

```diff
@@ -47,14 +47,25 @@ export default ({
     }
   },
   methods: {
+    // FIXME: probably needs to be defined centrally somewhere else
+    isMacSafari (userAgent) {
+      const safari = /Macintosh.*Safari/
+      const chrome = /Macintosh.*Chrome/
+      const edge = /Macintosh.*Edg/
+      return userAgent.match(safari) &&
+              !(userAgent.match(chrome) || userAgent.match(edge))
```

Comment:
A few questions here:

I am confused by this condition... why is it written this way instead of simplified to a single regex match?

Is it possible that `/Macintosh.*Safari/` and `/Macintosh.*Chrome/` can match at the same time? What are the actual user agent strings we're dealing with?

Reply:
I tested these browsers on my mac

Chrome
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36"

Edge
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.54 Safari/537.36 Edg/101.0.1210.39

Brave
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.54 Safari/537.36'

Firefox (no issue)
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:99.0) Gecko/20100101 Firefox/99.0"
​
Safari
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0 Safari/605.1.15"

Interesting article related to it 
https://blogs.windows.com/msedgedev/2015/06/17/building-a-more-interoperable-web-with-microsoft-edge/
Apparently using detection with userAgent as you are aware is rife with problems not the least due to spoofing

Reply:
Ah, I see. OK, yeah that's kinda crazy. Nice job 👍

Reply:
Can you write a comment with a few sentences explaining that so that it doesn't get forgotten?